fem/lininteg.cpp

```diff
@@ -93,6 +93,37 @@ void BoundaryLFIntegrator::AssembleRHSElementVect(
    }
 }
 
+void BoundaryLFIntegrator::AssembleRHSElementVect(
+   const FiniteElement &el, FaceElementTransformations &Tr, Vector &elvect)
+{
+   int dof = el.GetDof();
+
+   shape.SetSize(dof);        // vector of size dof
+   elvect.SetSize(dof);
+   elvect = 0.0;
+
+   const IntegrationRule *ir = IntRule;
+   if (ir == NULL)
+   {
+      int intorder = oa * el.GetOrder() + ob;  // <----------
```

Comment:
Please, either add a comment or remove the arrow. I am not sure what it means.

Reply:
I'm not sure either.  Many of the integrators have it but you're right.  I'll try to add a comment.

Reply:
Long time ago, I was adding these arrows to indicate a choice that may require some additional scrutiny. I think I used them only for the integration order formulas in various types of integrators.